CONTRIBUTING.md

```diff
@@ -19,18 +21,36 @@ Punchcard is a headless CMS designed with content strategy at its heart. With a
 
 * **DO NOT ISSUE A PULL REQUEST WITHOUT FIRST [SUBMITTING AN ISSUE](#submitting-issues)**
 * **ALL PULL REQUESTS MUST INCLUDE A [DEVELOPER CERTIFICATE OF ORIGIN](#developer-certificate-of-origin)**
+* Please follow our [Pull Request Guidelines](#issue-guidelines) when creating a new issue.
 * Pull requests should reference their related issues. If the pull request closes an issue, [please reference its closing from a commit messages](https://help.github.com/articles/closing-issues-via-commit-messages/). Pull requests not referencing any issues will be closed.
 * Commit messages _must_ begin with the [relevant emoji](#emoji-cheatsheet) describing what the commit does.
 * Pull request titles should be descriptive, explaining at the high level what it is doing, and should be written in the same style as [commit messages](#git-commit-messages).
-* Include a summarized list of changes, additions, and deletions in the body of the pull request in the same style as [commit messages](#git-commit-messages).
-* Follow our JavaScript and CSS styleguides. We have linters set up to catch most of it.
-* Ensure that [EditorConfig](http://editorconfig.org/) installed in the editor used to work on the site and that it is functioning properly.
-* Do not squash or rebase commits when submitting a Pull Request. It makes it much harder to follow work and make incremental changes.
+* Do not squash or rebase commits when submitting a Pull Request
 * Ensure no Emoji tags are used in the title of the Pull Request
 
+## Pull Request Guidelines
+
+In order to expedite the process of reviewing and merging pull requests, we request the following guidelines are followed.
+
+### Tips for Submitting Great PRs
+
+Beyond just writing the code needed to resolve an issue, there are a few best practices that should be followed to submit a great PR:
+
+* We have [EditorConfig](http://editorconfig.org/) files set up for Punchcard. Having EditorConfig installed and used will help ensure that editors are configured to follow our conventions.
+* We have JavaScript and Sass linting set up (where appropriate). Before committing code, make sure that linting has been run locally and passes by running `npm run lint`
+* We have automated testing set up. It can be run locally by by running `npm run test`. This will run linting, our test suite, and show code coverage.
```

Comment:
Further down this list I say that all required automated checks must pass, and I guess I imply this, but I can make it explicit if you'd prefer